src/lib/services/api-token-service.test.ts

```diff
@@ -111,7 +111,7 @@ test('Api token operations should all have events attached', async () => {
         (e) => e.type === API_TOKEN_UPDATED,
     );
     expect(updatedApiTokenEvents).toHaveLength(1);
-    expect(updatedApiTokenEvents[0].preData.expiresAt).toBeDefined();
+    expect(updatedApiTokenEvents[0].preData.expiresAt).toBeUndefined();
```

Comment:
When we first create the token the expiration is not provided. Probably this was working because of the fix in fake api token store